### PR TITLE
[[ Cleanup ]] Clean up a couple of bits of IDE code

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 6fceb0f

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 6fceb0fccd486fbbc670d5de0d8d21dc1fd5d32b

File: Toolset/palettes/tools/revtools.livecodescript [hunk 1]
```diff
@@ -138,16 +138,17 @@ on framePreferenceSelected pPreference, pValue
...
          put revIDEGetPreference("revTools_show") into tCurrentPreferenceValue
-         if pValue is among the items of tCurrentPreferenceValue then
-            repeat with x = the number of items of tCurrentPreferenceValue down to 1
-               if item x of tCurrentPreferenceValue is pValue then
-                  delete item x of tCurrentPreferenceValue
-               end if
-            end repeat
+         put lineOffset(pValue, tCurrentPreferenceValue) into tLineOffset
+         if tLineOffset is not 0 then
+            delete line tLineOffset of tCurrentPreferenceValue
```
Comment (livecodeali) [inline Toolset/palettes/tools/revtools.livecodescript]: Oops, I think this should be itemOffset not lineOffset

**Comment by montegoulding** [comment]
@livecode-vulcan review ok a1c4ab9

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok a1c4ab97c4d125db940f960fd03818510b144a57

**Pull request merged**